app/controllers/schools/advice/base_out_of_hours_controller.rb

```diff
@@ -27,10 +28,22 @@ def analysis_dates
           end_date: end_date,
           one_years_data: one_years_data?(start_date, end_date),
           recent_data: recent_data?(end_date),
-          months_analysed: months_analysed(start_date, end_date)
+          months_of_data: months_between(start_date, end_date),
+          last_full_week_start_date: last_full_week_start_date(end_date),
+          last_full_week_end_date: last_full_week_end_date(end_date),
         )
       end
 
+      #for charts that use the last full week
```

Comment:
maybe an additional comment here & below to say for all(?) charts we treat the beginning of the week as a Sunday and end of the week as a Saturday (hence the -1)